src/main/java/com/iemr/helpline104/data/comoOutbound/T_104CoMoOutboundCallDetails.java

```diff
@@ -59,12 +59,12 @@ public T_104CoMoOutboundCallDetails() {
 		// TODO Auto-generated constructor stub
 	}
 
-	public T_104CoMoOutboundCallDetails(Long iD, Integer uSRMappingID, String czentrixCallID, String callType,
+	public T_104CoMoOutboundCallDetails(Long iD, Integer uSRMappingID, String callCentreID, String callType,
```

Comment:
Please change this property to callId. It is not callCentreId. if you look at the original property, it was czentrixCallId.